tests/generate_fixtures/generate_target_anim_files.py

```diff
@@ -57,11 +59,14 @@
 comparisons = permutations([_.stem for _ in Path(INPUT_DIR).glob("*")], 2)
 inputs = {_.stem: _ for _ in Path(INPUT_DIR).glob("*")}
 
+nucmer = get_nucmer()
+print(f"Using nucmer {nucmer.version} at {nucmer.exe_path}")
```

Comment:
Related to the comment about ignoring T201 - I think it's better to explicitly `# noqa` the odd cases where we _want_ to `print()` than have a bunch of debug `print()`s come through. What are your thoughts?

Reply:
I'll swap over to ``# noqa: T201`` in place of the per-file-ignores. 

I was imagining the fixture generator scripts as being largely self-contained and less likely to ever be hooked into whatever logging framework the main code will use. i.e. Using ``print`` here was and is pragmatic.

Reply:
I was thinking that we might use `click` for those fixture generator scripts, which has `click.echo()`. It's not a huge priority, and `print()` is pragmatic, for now.